acceptance-tests/src/main/java/io/blueocean/ath/BasePage.java

```diff
@@ -0,0 +1,93 @@
+package io.blueocean.ath;
+
+import org.openqa.selenium.*;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.FluentWait;
+
+import javax.inject.Inject;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+/**
+ * Provides utility methods to downstream test classes
+ */
+public class BasePage {
+    @Inject
+    protected WebDriver driver;
+
+    @Inject @BaseUrl
+    protected String base;
+
+    public BasePage() {
+    }
+
+    public BasePage(WebDriver driver) {
+        this.driver = driver;
+    }
+
+    /**
+     * Gets the provided WebDriver instance
+     * @return from driver
+     */
+    protected WebDriver getDriver() {
+        return driver;
+    }
+
+    /**
+     * Gets the current browser URL
+     * @return current url
+     */
+    public String getCurrentUrl() {
+        return getDriver().getCurrentUrl();
+    }
+
+    /**
+     * Gets the browser url relative to the base
+     * @return relative url
+     */
+    public String getRelativeUrl() {
+        return getCurrentUrl().substring(base.length());
+    }
+
+    /**
+     * Navigates to a specified url
+     * @param url where to go
+     */
+    public void goTo(String url) {
+        getDriver().get(url);
+    }
+
+    /**
+     * Navigates to a relative url to the base url
+     * @param url where to go
+     */
+    public void go(String url) {
```

Comment:
I like the idea of two methods to navigate to relative and absolute paths, although it's not clear which one is which from the names `go` and `goTo`

Reply:
agreed, maybe could just inspect the url if it starts with `http(s)` then don't use the base